FrontEnd/src/features/container/trangchu/domestic-tour/DomesticTour.js

```diff
@@ -0,0 +1,158 @@
+import "./Tour.css";
```

Comment:
Please double check the position to import the css file. 
Suggest: At the end of import

Reply:
FYI: @khanhvan0502